src/sql/workbench/parts/query/browser/queryResultsView.ts

```diff
@@ -366,11 +365,13 @@ export class QueryResultsView extends Disposable {
 		}
 	}
 
-	public showResults() {
+	public showResults(show: boolean = true) {
 		if (!this._panelView.contains(this.resultsTab)) {
```

Comment:
You never use this with false so this seems like an unnecessary change (for now)